src/hooks/useFormHooks.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+
+interface IFormContents {
+  to: string;
+  message: string;
+}
+
+const useFormHooks = () => {
+  const [formData, setSetFormData] = useState<IFormContents>({
+    to: '',
+    message: '',
+  });
+
+  const inputOnChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (e.target.value.length < 8) {
+      setSetFormData((prev) => {
+        return { ...prev, to: e.target.value };
```

Comment:
이거 굳이 안에 return 한번더 정의한 이유는 머지? 

```suggestion
setSetFormData((prev) => { ...prev, to: e.target.value })
```

Reply:
return 전에 안에 데이터 확인한다고 해둔거였는데 미처 수정을 못했따.. 고마워!!!